### PR TITLE
change from very old HTML to Markdown

### PR DESCRIPTION
**Pull request by retokromer**
Make the To-Do easier to read. This in meant in preparation of https://github.com/emaculation/BasiliskII/issues/74.

### CODE REVIEW

**Comment by ianfixes** [comment]
Hahah 😄 I was scripting something up to import directly from the HTML, but this works too

**Comment by retokromer** [comment]
I have a BBEdit scripts that converts all up to HTML 4.0 and XHTML 1.0 into HTML 5.

File: src/MacOSX/TODO.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Floppy stuff. If a floppy is mounted by the OS X Finder, it is busy and cannot be opened by the emulator
+- Interrupt function for emulator.
+- 'nogui' to disable GUI alerts (and maybe preferences, but I need to split `MainMenu.nib` to do that).
+
+## Possible Enhancements
+- Use NSFileManager's movePath:toPath:handler to rename all a file's forks in `extfs_macosx.mm`.
+- Emulator snapshot - save the current emulator state (memory + registers) to a file for fast startup next time.
+- Multiple emulators. The window stuff is mostly there, but the uae_cpu code has lots of globals, and is not re-entrant
+- Real addressing mode for the emulator. Mike Sliczniak had done most of the work, but with real addressing the emulator just crashes for me.
+- Improve Objective-C object allocation (e.g. http://www.mulle-kybernetik.com/artikel/Optimisation/opti-5.html) – **Caveat:** On 2019-03-03 the link is broken. RK
```
Comment (ybart) [inline src/MacOSX/TODO.md:26]: It seems it's just a typo, this one works: http://www.mulle-kybernetik.com/artikel/Optimization/opti-5.html

Comment (retokromer) [reply src/MacOSX/TODO.md:26]: American English vs. British English

Comment (ianfixes) [reply src/MacOSX/TODO.md:26]: Nice, I've moved this to https://github.com/emaculation/macemu/issues/126

**Pull request merged**